### PR TITLE
elfutils (new formula)

### PR DESCRIPTION
**Pull request by iMichka**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/elfutils.rb [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      rm_f file
+      Pathname(file).write("exit 77", perm: 0755)
+    end
+
+    system "make", "check"
+    system "make", "install"
+  end
+
+  test do
+    output = `#{bin}/elfutils-nm #{bin}/elfutils-nm`
```
Comment (carlocab) [inline Formula/elfutils.rb:60]:
```suggestion
    output = shell_output("#{bin}/elfutils-nm #{bin}/elfutils-nm")
```

**Comment by iMichka** [comment]
> 55: col 5: Formulae in homebrew/core (except e.g. cryptography, libraries) should not run build-time checks

I'm going to remove `"make", "check"` then

**Comment by SMillerDev** [comment]
> configure: error: need libdebuginfod (or dummy), use --disable-debuginfod to disable.

File: Formula/elfutils.rb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+           "--disable-libdebuginfod",
+           "--program-prefix=elfutils-",
+           "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    output = shell_output("#{bin}/elfutils-nm #{bin}/elfutils-nm")
+    assert_match "elf_kind", output
```
Comment (MikeMcQuaid) [inline Formula/elfutils.rb:30]:
```suggestion
    assert_match "elf_kind", shell_output("#{bin}/elfutils-nm #{bin}/elfutils-nm")
```

**Review by MikeMcQuaid — DISMISSED** [review]

**Review by carlocab — APPROVED** [review]

**Pull request merged**